packages/tasks/src/local-apps.ts

```diff
@@ -1,6 +1,12 @@
 import type { ModelData } from "./model-data";
 import type { PipelineType } from "./pipelines";
 
+interface Snippet {
+	title: string;
+	setup: string;
```

Comment:
```suggestion
	setup?: string;
```

make the setup step optional. Maybe some snippets will not need the setup step

Reply:
Fixed in https://github.com/huggingface/huggingface.js/pull/778/commits/7983478d5f136b2e101752b229fa906f06c719f1